Game_theLastBattle/Fighters.swift

```diff
@@ -25,20 +26,33 @@ protocol Guild {
 
     var dead: Bool {get set} // Is the character dead?
     
-//    func description() -> String // Details of the wizard
+    var description: String {get set} // Details of the fighter
+    
+}
+
+
+
+
+
+// -------------------------------------------------------------------------------------------------------
+// MARK: Kind of Fighters
+// >>> Gives its properties to the character
+// -------------------------------------------------------------------------------------------------------
 
+enum Kind: String {
+    case wizard, warrior, dwarf
 }
 
 
 
 
 
 // -----------------------------------------------------------------------------------------------------
-// MARK: Class Characters
+// MARK: Class Fighter
 // >>> General characteristics of characters
 // -----------------------------------------------------------------------------------------------------
 
-class Characters {
+class Fighter: FighterInTheGuild {
```

Comment:
Tu pourrais renommer Fighters en Fighter, au singulier, comme tu definis les caracteristiques d'un seul fighter.

Reply:
Ok merci, c'est noté.
Je viens de faire des modifs sur la façon dont sera gérée le naming des fighters, 
via l'initialisateur  du fighter pour l'intégration, ensuite je vais créer une autre fonction au niveau de la class Player (ou Game) qui vérifiera l'unicité du nom.
Est-ce que pour toi, cette piste peut fonctionner ?
Merci a+
Greg